packages/apollo-cache-control/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+### Unreleased
+
+* Add support for `inherited` key to skip setting a `maxAge` to 0 if a
```

Comment:
@lunks I wonder if it would provide more utility for inherit to be recursive, such that subfields will also inherit their cacheability as well from grandparents, etc. Basically, should `inherit` itself be inherited?

Reply:
If I understand correctly what you're proposing, I don't think that's a good idea, because it would mark an entire tree as cacheable. And I think objects should always have to opt-in to caching (even if that's through `inherit`).

Reply:
For a bit of background on why I think inheritance should be opt-in, see here:
https://github.com/apollographql/apollo-cache-control-js/issues/14#issuecomment-37315847.6